accounts/abi/abi_test.go

```diff
@@ -75,9 +75,24 @@ func TestReader(t *testing.T) {
 	}
 
 	// deep equal fails for some reason
-	t.Skip()
-	if !reflect.DeepEqual(abi, exp) {
-		t.Errorf("\nabi: %v\ndoes not match exp: %v", abi, exp)
+	for name, expM := range exp.Methods {
+		gotM, exist := abi.Methods[name]
+		if !exist {
+			t.Errorf("Missing expected method %v", name)
+		}
+		if !reflect.DeepEqual(gotM, expM) {
+			t.Errorf("\nGot abi method: \n%v\ndoes not match expected method\n%v", gotM, expM)
+		}
```

Comment:
How about using [testify](https://godoc.org/github.com/stretchr/testify)?

6 lines above could be substituted with:
```
require.True
require.Equal
```

Also this solves another issue: if the first `if` fails then the second one shouldn't be executed.

Reply:
I haven't gotten used to testify yet, that's the reason :)
I'll keep it in mind for future reference